core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/html/libraries/iframe-edit-mode.css.ts

```diff
@@ -6,7 +6,10 @@ const mdShadow3 = '0 10px 20px rgba(0, 0, 0, 0.19), 0 6px 6px rgba(0, 0, 0, 0.23
 const white = '#fff';
 const grayLight = '#c5c5c5';
 
-export const getEditPageCss = (timestampId: string): string => {
+export const getEditPageCss = (
+    timestampId: string,
+    origin: string = globalThis.location.origin
```

Comment:
Should we use `window` since we are in a browser environment? It can be just `location.origin` too.

Reply:
@zJaaal You forgot to push ahh!

Reply:
Done aaah!